test/Tizen.NUI.Samples/Tizen.NUI.Samples/Samples/SubWindowTest.cs

```diff
@@ -109,144 +149,142 @@ public void WinTouchEvent(object sender, Window.TouchEventArgs e)
         {
-            log.Fatal(tag, $"test 3 : 1) create sub window2 which doesn't have any setting 2) dispose it after 3 second delay");
-            subWin2 = null;
-            subWin2 = new Window();
-            tm = new Timer(3000);
-            tm.Tick += Tm_Tick;
-            tm.Start();
+            subWinSecond = null;
+            subWinSecond = new Window();
+            timer = new Timer(3 * oneSecond);
+            timer.Tick += TimerTick;
+            timer.Start();
         }
-        void TestCase4()
+
+        private void TestCreateSubWindowSecondWhichHasSomeSettingAndDisposeItAfterThreeSecondsDelay()
         {
-            log.Fatal(tag, $"test 4 : 1) create sub window2 which has some setting 2) dispose it after 3 second delay");
-            subWin2 = null;
-            subWin2 = new Window("subWin2", new Rectangle(100, 100, 100, 100), false);
-            subWin2.BackgroundColor = Color.Red;
-            tm = new Timer(3000);
-            tm.Tick += Tm_Tick;
-            tm.Start();
+            subWinSecond = null;
+            subWinSecond = new Window("subWinSecond", new Rectangle(testPosition, testPosition, testSize, testSize), false)
+            {
+                BackgroundColor = Color.Red
+            };
+            timer = new Timer(3 * oneSecond);
+            timer.Tick += TimerTick;
+            timer.Start();
         }
-        void TestCase5()
+
+        private void TestCreateSubWindowSecondWhichHasSomeSettingAndAddSomeDummyObjectAndDisposeItAfterThreeSecondsDelay()
         {
-            log.Fatal(tag, $"test 5 : 1) create sub window2 which has some setting 2) add some dummy object 3) dispose it after 3 second delay");
-            subWin2 = null;
-            subWin2 = new Window("subWin2", new Rectangle(500, 500, 300, 300), false);
-            subWin2.BackgroundColor = Color.Red;
-            View v1 = new View()
+            subWinSecond = null;
+            subWinSecond = new Window("subWinSecond", new Rectangle(5 * testPosition, 5 * testPosition, 3 * testSize, 3 * testSize), false)
+            {
+                BackgroundColor = Color.Red
+            };
+            View testView = new View()
             {
-                Size = new Size(50, 50),
-                Position = new Position(50, 50),
+                Size = new Size(testSize, testSize),
+                Position = new Position(testPosition, testPosition),
                 BackgroundColor = Color.Yellow,
             };
-            subWin2.Add(v1);
+            subWinSecond.Add(testView);
 
-            tm = new Timer(3000);
-            tm.Tick += Tm_Tick;
-            tm.Start();
+            timer = new Timer(3 * oneSecond);
+            timer.Tick += TimerTick;
+            timer.Start();
         }
 
-        private bool Tm_Tick(object source, Timer.TickEventArgs e)
+        private bool TimerTick(object source, Timer.TickEventArgs e)
```

Comment:
I suggest name DisposeTimer or DisposeSubWinSecondTimer to know what this timer should be supposed to do.

Reply:
Thank for review. I fixed it.